datafusion/core/src/config.rs

```diff
@@ -87,6 +108,13 @@ impl BuiltInConfigs {
                 filter can add additional overhead when the file format does not fully support \
                 predicate push down.",
                 false,
+            ),
+            ConfigDefinition::new_u32(
+            OPT_BATCH_SIZE,
+            "Default batch size while creating new batches, it's especially useful for \
```

Comment:
I think this description can be improved but that seemed outside the scope of this PR